CHANGELOG.md

```diff
@@ -1,13 +1,50 @@
 # Changelog
 
-## [4.2.1] - 2019-10-29
+## [4.3.0] - 2020-01-14
+
+### Added
+ - Badge for numbers added
```

Comment:
I think it would look better without the `added` as long as we're in the the `added` list